src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java

```diff
@@ -50,6 +50,10 @@ public void testToEntity() {
         assertEquals(expectedQuestion.getShowGiverNameTo(), actualQuestion.getShowGiverNameTo());
         assertEquals(expectedQuestion.getShowRecipientNameTo(), actualQuestion.getShowRecipientNameTo());
         assertEquals(expectedQuestion.getShowResponsesTo(), actualQuestion.getShowResponsesTo());
+        assertNotNull(expectedQuestion.getCreatedAt());
+        assertNotNull(actualQuestion.getCreatedAt());
+        assertNotNull(expectedQuestion.getUpdatedAt());
+        assertNotNull(actualQuestion.getUpdatedAt());
         assertEquals(expectedQuestion.getCreatedAt(), actualQuestion.getCreatedAt());
         assertEquals(expectedQuestion.getUpdatedAt(), actualQuestion.getUpdatedAt());
```

Comment:
And this?

Reply:
How did you test the code? Based on my tests ,the program passed, so I would like to test like you do.

Reply:
@Barbado2017 the idea is to replace those two lines with the assertNotNull checks. If the test execution is slow those two lines will fail which makes it unstable.